### PR TITLE
UPD: Update gwyddion to 2.55

### PR DESCRIPTION
**Pull request by rowue**
#### Description

This pull request introduces an update for gwyddion to 2.55

###### Type(s)

- [ ] bugfix
- [ X] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6
Xcode 10.1

###### Verification 
Have you

- [ X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ X] checked your Portfile with `port lint`?
- [X ] tried a full install with `sudo port -vst install`?
- [X ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Please rebase your branch here to remove the merge commit, and also your commit message does not follow our guidelines. Please update it to the form ‘portname: description’.

File: science/gwyddion/Portfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ configure.args      --disable-desktop-file-update \
                     --x-includes=${prefix}/include \
                     --x-libraries=${prefix}/lib
 
-patchfiles          patch-libgwyddion-gwyutils-c.diff
+# patchfiles          patch-libgwyddion-gwyutils-c.diff
```
Comment (pmetzger) [inline science/gwyddion/Portfile]: If the patchfile isn't needed any more, could we remove it (and this line as well?)

**Comment by pmetzger** [comment]
@rowue if you don't know how to do what @cjones051073 has asked, let us know and we can help you with it.

File: science/gwyddion/Portfile (hunk 1 above)
Comment (rowue) [reply science/gwyddion/Portfile]: done

**Comment by rowue** [comment]
If I get it right, everything should be in order now.

Best,

   rowue

**Comment by pmetzger** [comment]
> If I get it right, everything should be in order now.

I believe the patch file is still present. However, I'm inclined to commit this and just remove it myself.

**Comment by rowue** [comment]
> > If I get it right, everything should be in order now.
> 
> I believe the patch file is still present. However, I'm inclined to commit this and just remove it myself.
removed

**Comment by pmetzger** [comment]
Merged.

**Pull request merged**